src/components/Dashboard/TodoList.vue

```diff
@@ -344,6 +352,10 @@
                     <template v-else-if="isTypeRegistration(item) && item.isEmptyFiling">
                       <span>Register using this NR</span>
                     </template>
+                    <template v-else-if="EnumUtilities.isTypeContinuationInApplication(item) && item.isEmptyFiling">
+                      <span v-if="getNameRequest">Continue In using this NR</span>
+                      <span v-else>Continue In as a Numbered Company</span>
+                    </template>
```

Comment:
Looks good.

How about changing isTypeAmalgamation, isTypeIncorporationApplication and isTypeRegistration to use the utilities (instead of the enum) for consistency here?